core/kernel/src/main/java/io/thorntail/logging/impl/MessageOffsets.java

```diff
@@ -4,23 +4,24 @@
  * @author Ken Finnigan
  */
 public interface MessageOffsets {
```

Comment:
Please use `git add -p` instead of `git add .` in future.

Reply:
Yeah, my bad

Reply:
No worries. It happens, we've all been there. :)